quarto/_tab-strata.qmd

```diff
@@ -14,7 +21,12 @@ plot_bmi_categories(plot_data = plot_data_latest) |>
   layout(legend = list(orientation = "h"))
 ```
 
-### Weekly change in BMI
+<h2 style="color: #2c3e50; margin-bottom: 1rem;">
+  Weekly trends in BMI
+</h2>
+
+The below chart shows the weekly distribution of BMI categories (underweight, normal, overweight, and obese) across different dates. Each bar represents the proportion of participants measured in that week, with stronger colours indicating weeks where more data was collected (i.e., more study participants logged updated weight measurements).
```

Comment:
```suggestion
The below chart shows the weekly distribution of BMI categories (underweight, normal, overweight, and obese) across different dates. Each bar represents the proportion of participants measured in that week. The strength of the colour indicates the volume of data collected: lighter for fewer participants and darker for more participants.
```

Reply:
This is describing the same alpha gradient as in other plot text descriptions, so maybe worth keeping the same phrasing throughout for consistency.